Makefile

```diff
@@ -1,10 +1,9 @@
 
-
 build:
-	docker build -t firefox .
+	docker build --build-arg FIREFOX_VERSION=$(shell cat VERSION) -t firefox:$(shell cat VERSION) .
```

Comment:
@SolidNerd Got you. But there is no way to pass a custom `FIREFOX_VERSION` using docker.io, right?
If so, I'd drop the "build arg" altogether and create a directory for each major Firefox version.
WDYT?